### PR TITLE
Add documentation about .font-weight-bolder/lighter

### PR DESCRIPTION
**Pull request by ysds**
`.font-weight-bolder` and `.font-weight-lighter` have been added by #26580,  but no documentation.

### CODE REVIEW

File: site/docs/4.1/utilities/text.md [hunk 1]
```diff
@@ -85,8 +85,10 @@ Quickly change the weight (boldness) of text or italicize text.
 
 {% capture example %}
 <p class="font-weight-bold">Bold text.</p>
+<p class="font-weight-bolder">Relative bolder than the parent element.</p>
```
Comment (XhmikosR) [inline site/docs/4.1/utilities/text.md]: You mean **compared to** the parent element?

File: site/docs/4.1/utilities/text.md [hunk 2]
```diff
@@ -85,8 +85,10 @@ Quickly change the weight (boldness) of text or italicize text.
...
 <p class="font-weight-normal">Normal weight text.</p>
 <p class="font-weight-light">Light weight text.</p>
+<p class="font-weight-lighter">Relative lighter than the parent element.</p>
```
Comment (XhmikosR) [inline site/docs/4.1/utilities/text.md]: Maybe we should add a note that this depends on the system font and the effect might not be visible on some systems.

For example on Windows with Firefox I get:

Comment (ysds) [reply site/docs/4.1/utilities/text.md]: That screenshot is the expected result. That parent is `.bd-example` (the font weight are inherited from `body`)

Comment (XhmikosR) [reply site/docs/4.1/utilities/text.md]: Yeah, that might need a note or, not.

**Comment by XhmikosR** [comment]
I'm not a native English speaker, so my edit might not be 100% correct.

**Comment by XhmikosR** [comment]
For reference: https://developer.mozilla.org/en-US/docs/Web/CSS/font-weight#Values

File: site/docs/4.1/utilities/text.md (hunk 2 above)
Comment (MartijnCuppens) [reply site/docs/4.1/utilities/text.md]: Maybe we should just use the same descriptions as in the mozilla docs?
image

Comment (XhmikosR) [reply site/docs/4.1/utilities/text.md]: In that case, we should probably link to the MDN doc as a reference.

**Review by mdo — APPROVED** [review]

**Review by blackc03r — COMMENTED** [review]
CMS-1691-P

**Pull request merged**